Adaptors/MongoDB/src/TaskTable.cs

```diff
@@ -229,7 +229,7 @@ public class TaskTable : ITaskTable
           throw new TaskNotFoundException($"Task {taskId} not found");
         }
 
-        throw new ArmoniKException($"Task already in a terminal state - {taskStatus.Single()} to {TaskStatus.Processing}");
+        throw new ArmoniKException($"Task was not acquired - {taskStatus.Single()} to {TaskStatus.Processing}");
```

Comment:
This is not AcquireTask but StartTask.

Reply:
But the root cause was that it was not acquired